### ISSUE TITLE
STA9750 Chloe384 MiniProject #01

### ISSUE THREAD

**Issue by Chloe384** [Chloe384/STA9750-2024-FALL#3]
Hi @michaelweylandt!

I've uploaded my work for MiniProject #01 - check it out!

https://Chloe384.github.io/STA9750-2024-FALL/mp01.html

Best,
Chloe

**Comment by michaelweylandt** [comment]
Thanks @Chloe384 !

Peer Feedback:

- @aaronkcooper
- @clintavarghese
- @doreenmurekerisoni
- @fatima8808

It is now time for the peer feedback round for Mini-Project #01. Please
review @Chloe384's submission for this mini-project and provide peer feedback.

Using the rubric at https://michael-weylandt.com/STA9750/miniprojects/mini01.html#rubric,
please grade their submission out of a total of 50 points.

For each of the five categories, please give them a separate score and provide
a total (sum) score across the entire assignment. Feel free to assign extra
credit if you feel it is warranted (following the rubric).

If you give a score of less than 5 for any category, please provide a suggestion
for improvement. (You can also give suggestions for any element they did well -
more feedback is always great!)

As you go through this peer feedback exercise, think about what you particularly
like about this submission and how you can incorporate that approach in your future
work. If something is particularly insightful or creative, give some kudos!

Evaluators: This should take you around 15 minutes per peer feedback. You are
not required to engage in substantial back-and-forth with @Chloe384, but you are of course
welcome to initiate a discussion.

@Chloe384: please engage fully with your peers. They are here to help you!

Submission URL should be: https://Chloe384.github.io/STA9750-2024-FALL/mp01.html

Feel free to link to other repos, the course documentation, or other useful
examples.

Thanks!
@michaelweylandt

CC: @charles-ramirez

**Comment by clintavarghese** [comment]
@Chloe384 Overall great project. I love your tables. 
Here are my scores
Communication: 9.5
Project Skelton: 10
Formatting & Display: 10
Code Quality: 10
Total: 49.5/50
[ Task4: Q2,  As you mentioned, UPT/VRM efficiency is determined based on the ratio, the smaller the ratio, the higher the efficiency. YOu have written the other way around. I would suggest you to update the result analyses]

**Comment by fatima8808** [comment]
Hi @Chloe384, great job completing the first mini project for our course! I like how organized your overall report was and clearly identified the tasks you've completed.

Written Communication: 9/10
You outlined your report well and made it easy to understand the objectives and answers to the questions. One suggestion you may be interested in is including a deeper analysis of your findings to generate more motivation in your report.

Project Skeleton: 10/10

Formatting & Display: 10/10
You did a nice job creating your tables and ensuring to include only necessary columns so there isn't information overload for the readers.

Code Quality: 10/10

Data Preparation: 10/10

Total Score: 49/50

**Comment by aaronkcooper** [comment]
Hi @Chloe384  ,
Really nice job on the first Mini-Project! Here is my breakdown of feedback:
**Written Communication: 9/10**
Overall the project flowed very well and was very easy to follow. Nice analysis on some of the questions but some tasks I felt could have used a bit more of context behind the motivation of looking at for example the change in ridership from 2019-2020 (COVID-19)

**Project Skeleton: 10/10**
All questions were answered as required.

**Formatting and Display: 10/10**
Tables were great and concise. Code blocks were effective and the reader isn't left a huge section at once to try to make sense of

**Code Quality: 10/10**
Code is great, I found no issues here.

**Data Preparation: 10/10**

**Total Score: 49/50**
Great Job!

**Comment by charles-ramirez** [comment]
Hi @doreenmurekerisoni, 

This is just a reminder that your peer review is due today at 11:45 pm.

**Comment by doreenmurekerisoni** [comment]
Hi @Chloe384, 

This is a great mini project #1 !!

Written Communication-9/10 I enjoyed going over your mini project #1. I was able to analyze your project with ease as your context was clear and well written.
Project Skeleton- 10/10 
Formatting and Display-10/10 Your tables were structed well and you used all neccesary details to make the information clear and concise to all readers.
Code Quality-10/10
Data Preparation-10/10

Total Score:49/50
